test_team_sei.py

```diff
@@ -22,7 +22,23 @@ def test_find_us_ssn(self):
     def test_find_email(self):
         results_list = find_email('My email address is jim.jones@jones.com')
```

Comment:
The ^ in your regex will only match at the beginning of the string.  You need to remove the ^ in order to match the email in this string.
```
 re.findall(r'^[a-zA-Z0-9+_.-]+@[a-zA-Z0-9.-]+', text)
```